api/v1/models/index.js

```diff
@@ -8,4 +9,10 @@ module.exports = {
 	Volunteer: require('./Volunteer'),
 	Team: require('./Team'),
 	TeamsUser: require('./TeamsUser')
+=======
```

Comment:
It definitely shouldn't run. The only thing I can think of is that I didn't add it to my commit